app/views/passengers/show.haml

```diff
@@ -31,11 +31,20 @@
   %td.yes= yes_no_image @passenger.permanent
 - if @passenger.temporary?
   %p
-    %b Doctor's Note Expiration Date:
-    = @passenger.expiration_display
+    %b Has Doctor's Note?
+    %td.yes= yes_no_image @passenger.doctors_note.present?
```

Comment:
Image? why?

Reply:
I didn't name that method. Also, it's an image, so.

Reply:
:neckbeard: Well, actually, it's a font glyph.

Reply:
Which is a pictoral representation of a character. Everything we look at is technically an image. Fight me

Reply:
So changing the method name could be done, it's not a great method name, but if we go down that rabbit hole with this PR it'll never end.

Reply:
I guess what I really meant to ask was, what does it look like? Why is that better than basic text? But I digress.